lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -1230,6 +1230,8 @@ void ThreadContext::AddBuiltInPropertyRecord(const Js::PropertyRecord *propertyR
 
 BOOL ThreadContext::IsNumericPropertyId(Js::PropertyId propertyId, uint32* value)
 {
+    if (Js::IsInternalPropertyId(propertyId)) return false;
```

Comment:
nit : please have brace even for a single statement body.
Also curious why this function's signature has BOOL even though we return bool (true/false).